bin/adhoc-scripts/injectUnified.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+"""
+This script can be used to create a default Unified configuration in central CouchDB
+"""
+
+import http.client
+import json
+import os
+import sys
+from pprint import pformat
+
+DOC = {"tiers_to_DDM": {"value": ["LHE", "GEN-SIM-DIGI-RAW-MINIAOD", "AODSIM", "MINIAODSIM", "GEN-SIM-RAW",
+                                  "GEN-SIM-RECO", "GEN-SIM-RECODEBUG", "AOD", "RECO", "MINIAOD", "ALCARECO",
+                                  "USER", "RAW-RECO", "RAWAODSIM", "NANOAOD", "NANOAODSIM", "FEVT", "PREMIX",
+                                  "GEN-SIM-DIGI-RAW-HLTDEBUG-RECO", "FEVTDEBUGHLT"],
+                        "description": "Datatiers that have a final MSOutput data placement"},
+       "tiers_no_DDM": {"value": ["GEN-SIM", "GEN", "SIM", "DQM", "DQMIO", "GEN-SIM-DIGI-RAW", "RAW",
+                                  "GEN-SIM-DIGI", "GEN-SIM-DIGI-RAW-HLTDEBUG"],
+                        "description": "Datatiers not meant to have a final MSOutput data placement"},
+       "tiers_with_no_custodial": {"value": ["DQM", "RECO", "RAWAODSIM", "FEVTDEBUGHLT"],
+                                   "description": "Datatiers not meant to have a Tape final MSOutput data placement. Can be overriden at campaign level"}
+       }
+
+if __name__ == '__main__':
+    url = 'cmsweb-testbed.cern.ch'
+    headers = {"Content-type": "application/json", "Accept": "application/json"}
+    encodedParams = json.dumps(DOC)
+
+    print(f"Updating unified configuration in {url} with content: {pformat(DOC)}")
+    conn = http.client.HTTPSConnection(url,
+                                       cert_file=os.getenv('X509_USER_PROXY'),
```

Comment:
before making a call, please add block to check this env parameters, and if they are empty please provide meaningful message. The script should exit with non zero status in this case.